spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -91,6 +91,99 @@
         end
       end
 
+      context 'reuses valid capture app urls when appropriate', allow_browser_log: true do
+        before do
+          allow(IdentityConfig.store).to receive(:doc_auth_max_attempts).and_return(max_attempts)
+          (max_attempts - 1).times do
+            socure_docv_upload_documents(docv_transaction_token: @docv_transaction_token)
+          end
+        end
+
+        context 'successfully erases capture app url when flow is complete' do
+          before do
+            DocAuth::Mock::DocAuthMockClient.reset!
+          end
+
+          it 'proceeds to the next page with valid info' do
+            document_capture_session = DocumentCaptureSession.find_by(user_id: @user.id)
+            expect(page).to have_current_path(fake_socure_document_capture_app_url)
+            visit idv_socure_document_capture_path
+            expect(page).to have_current_path(idv_socure_document_capture_path)
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).
+              to eq(fake_socure_document_capture_app_url)
+            socure_docv_upload_documents(
+              docv_transaction_token: @docv_transaction_token,
+            )
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).to be_nil
+          end
+
+          it 'submits front ID, exits app, reuse capture app url' do
```

Comment:
we only have 1 capture app url ... how do we know that the url is not changing from one url to another when the value assigned in our spec is always the same? 🤔